contrib/ci/fwupd_setup_helpers.py

```diff
@@ -169,16 +169,33 @@ def _get_installer_cmd(os, yes):
     return installer
 
 
+def _get_update_cmd(os, yes):
+    if os == "debian" or os == "ubuntu":
+        updater = ["apt", "update"]
+    elif os == "fedora":
+        updater = ["dnf", "update"]
```

Comment:
If you just want to update the package metadata on fedora, you want `dnf makecache` -- `dnf update` updates the metadata *and* updates all the packages with updates in one step.

Reply:
@superm1 punt to 1.8.12 too?

Reply:
let's punt this one